float64 mapWidth
float64 mapHeight
float64 mapCenterX
float64 mapCenterY
mower_map/MapArea[] navigationAreas
mower_map/MapArea[] mowingAreas